packages/framework/presence/src/presenceDatastoreManager.ts

```diff
@@ -319,10 +319,19 @@ export class PresenceDatastoreManagerImpl implements PresenceDatastoreManager {
 			// list across all clients. We need something to provide suggested order
 			// to prevent a flood of broadcasts.
 			const quorumMembers = this.runtime.getQuorum().getMembers();
-			const indexOfSelf =
-				quorumMembers.get(clientId)?.sequenceNumber ??
+			let indexOfSelf;
+			const self = quorumMembers.get(clientId);
+			if (self) {
+				indexOfSelf = 0;
```

Comment:
Changing the name of indexOfSelf to something else would be helpful if the intended purpose is to find unique broadcast slot? Why is the change better the previous approach - is it to force uniqueness and avoid client overlapping on broadcast? This description would be helpful in PR description.